### ISSUE TITLE
Is there a way in InstaRaider to download all photos associated with a given tag

### ISSUE THREAD

**Issue by AntreasAntoniou** [akurtovic/InstaRaider#45]
I need a lot of data associated with a specific task, basically the photos, likes and comments of each photo. Is there a quick way to do this in InstaRaider, or should I try something else. Thanks for taking the time to read this.

**Comment by akurtovic** [comment]
@AntreasAntoniou The short answer is no. The code could easily be modified to pass a tag as a parameter instead of a username. InstaRaider would then load https://www.instagram.com/explore/tags/YOURTAG instead of the profile page, but I assume some modifications would have to be made in regards to the selectors for loading the page and getting the image URLs, as well as refactoring some of the code involved with parsing the user profile page for how many images the user has, etc. It wouldn't be hard to do, but it's not supported right now.

**Comment by Seirrah** [comment]
This is something I'm after too.

**Comment by AntreasAntoniou** [comment]
I found a way to do it. However instagram was limiting me to the number of images per page, even if I scrolled down I wouldn't get new pictures. BUT I used an API call to get tags similar to my own and then recursively get more and more similar tags. Each tag gives you 33 images, 24 random + 9 top ranking. So I did something like a few 5 steps of recursion and got 3000 tags = 33 x 3000 = 99000 images. Good enough for my deep learning model.
